consensus/votepower/roster.go

```diff
@@ -24,10 +25,22 @@ var (
 
 // Ballot is a vote cast by a validator
 type Ballot struct {
-	SignerPubKey      shard.BlsPublicKey `json:"bls-public-key"`
-	BlockLeader       shard.BlsPublicKey `json:"leader-when-signed"`
-	BlockHeightHeight uint64             `json:"block-height"`
-	Signature         *bls.Sign          `json:"signature"`
+	SignerPubKey shard.BlsPublicKey `json:"bls-public-key"`
+	BlockLeader  shard.BlsPublicKey `json:"leader-when-signed"`
```

Comment:
on a second thought, actually using viewID will be an easier way to distinguish signing blocks from different leaders.

Reply:
but will viewID from various shards be meaningful to beaconchain?